tests/test_capturehandler.py

```diff
@@ -83,6 +83,8 @@ class TestCaptureHandler(TestGramex):
 
     @classmethod
     def setupClass(cls):
+        if not paths['phantomjs']:
+            raise SkipTest('phantomjs is not installed')
```

Comment:
See if you can use `unittest.skipUnless` as a decorator for the class.

Reply:
I wasn't able to get `unittest.skipUnless` to work because

- `TestCaptureHandler` is the base class for `TestCaptureHandlerChrome`. If I skipUnless the first, it skips the second
- It still runs setupClass, though the [docs](https://docs.python.org/3/library/unittest.html) say that it won't. Might be a unittest - nosetests compatibility issue

I'll add a comment for now and proceed.